src/explore-education-statistics-admin/src/components/form/FormFieldThemeSelect.tsx

```diff
@@ -0,0 +1,46 @@
+import FormThemeSelect, {
```

Comment:
following comment in `src/explore-education-statistics-admin/src/pages/publication/components/PublicationDetailsForm.tsx`

We can remove this file I believe.